src/QirRuntime/test/Standalone/InputTypes/qir-driver.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include "../Shared/CLI11.hpp"
+
+// Can manually add calls to DebugLog in the ll files for debugging.
+extern "C" void DebugLog(int64_t value)
+{
+    std::cout << value << std::endl;
+}
+extern "C" void DebugLogPtr(char* value)
+{
+    std::cout << (const void*)value << std::endl;
+}
+
+extern "C" void SetupQirToRunOnFullStateSimulator();
+extern "C" void Microsoft__Quantum__Testing__QIR__InputTypes__body( // NOLINT
+    int64_t anInt,
+    double_t aDouble,
+    bool aBool,
+    char* aString,
+    int64_t anArrayLength,
+    int64_t* anArray);
+
+int main(int argc, char *argv[])
+{
+    try{
```

Comment:
It looks like you have a mix of same-line and next-line brace styles; which is used in this codebase?

Reply:
The convention is next-line brace styles. I'll change it to make it consistent.

Reply:
FYI: if you run clang formatter, it will apply the style automatically

Reply:
Ran clang formatter :)